docs/source/platform/federation.mdx

```diff
@@ -0,0 +1,208 @@
+
+For instance, in order to have a canary deployment, you might maintain two production graphs in the [Schema Registry](https://www.apollographql.com/docs/platform/schema-registry), one called `prod` and one called `prod-canary`. Your deployment of a change to some implementing service named "foo" might look something like this:
+
+1. Check changes in "foo" against `prod` and `prod-canary`: `apollo service:check --tag=prod --serviceName=foo && apollo service:check --tag=prod-canary --serviceName=foo`
+1. Deploy changes to "foo" into your production environment (_Note: This will not roll out changes to the gateway yet_)
+1. Roll over the `prod-canary` graph, containing one gateway container, using `apollo service:push --tag=prod-canary --serviceName=foo` (_Note: If composition fails due to intermediate changes to the canary graph, new configuration will not be rolled out_)
+1. Wait for health checks to pass against the canary, watch dashboards, etc.
+1. After the canary is stable, roll out the changes to the rest of production using `apollo service:push --tag=prod --serviceName=foo`
+
+Because you can [tag metrics with variants](https://www.apollographql.com/docs/platform/schema-registry/#associating-metrics-with-a-variant) as well, you can use [Apollo Graph Manager](https://engine.apollographql.com) to verify a canary's performance before rolling out changes to the rest of the graph. You can also use a similar strategy with variants to support a variety of other advanced deployment workflows, like blue/green deployments.
+
+## Managed Federation
+[//]: # (Description: This section should discuss the basic idea of managed federation without getting into specific and talk vaguely of the problem of service rollout and looking for a balance of automation and observability, workflow, etc.)
+[//]: # (Assignee: Jackson)
+[//]: # (Reviewer: Adam)
+
+In small scale cases it can be sufficient to define the list of services comprising a federated graph directly within the gateway, and manually update the list and restart the server as services are modified and created. However for larger scale projects it is helpful to be able to remotely manage the gateway, such that new services may be added, or existing services modified, all with zero downtime.
+
+For these use cases, a gateway can be configured to pull a remotely hosted service list from the [Schema Registry](https://www.apollographql.com/docs/platform/schema-registry/) on `Apollo Dashboard`. This is `Managed Federation`, and it is a key component of building a robust federated graph that can be concurrently developed by teams of any size.
+
+#### Overview
+
+// TODO: Document the overview of managing federation workflow, happy path
```

Comment:
@zionts 	Is this unassigned section removable? The content seems to be in the "Managed Federation" section already?